src/app/components/cards/cards.component.scss

```diff
@@ -0,0 +1,66 @@
+@import "../../../styles/variables.scss";
+
+.c-title {
+    color: $color-black;
+    font-size: $font-h3;
+    font-weight: $font-bold;
+    margin-bottom: 30px;
+}
+
+.c-filters {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    height: 60px;
+    &__left {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        text-align: center;
```

Comment:
There is no need for `text-align: center`; if you use `justify-content: center;`